python/lsst/obs/decam/ingest.py

```diff
@@ -123,23 +139,49 @@ def getInfo(self, filename):
         The user creates the registry by running
 
           ingestImagesDecam.py outputRepository --mode=link instcal/*fits
-        """
-        if self.expnumMapper is None:
-            self.buildExpnumMapper(os.path.dirname(os.path.abspath(filename)))
-
-        # Note that phuInfo will have
-        #   'side': 'X', 'ccd': 0
-        phuInfo, infoList = super(DecamParseTask, self).getInfo(filename)
-        expnum = phuInfo["visit"]
-        phuInfo[self.instcalPrefix] = self.expnumMapper[expnum][self.instcalPrefix]
-        phuInfo[self.dqmaskPrefix]  = self.expnumMapper[expnum][self.dqmaskPrefix]
-        phuInfo[self.wtmapPrefix]   = self.expnumMapper[expnum][self.wtmapPrefix]
-        for idx, info in enumerate(infoList):
-            expnum = info["visit"]
-            info[self.instcalPrefix] = self.expnumMapper[expnum][self.instcalPrefix]
-            info[self.dqmaskPrefix]  = self.expnumMapper[expnum][self.dqmaskPrefix]
-            info[self.wtmapPrefix]   = self.expnumMapper[expnum][self.wtmapPrefix]
 
+        For raw data, the registry can be created by running
+
+          ingestImagesDecam.py outputRepository --mode=link --filetype="raw" raw/*.fits.fz
```

Comment:
It makes me a little nervous to have this documentation here since it can get out of sync (which you caught this time, but maybe the next person won't).  Would it be sufficient to point to the ingest script documentation from here?